src/System.Security.Cryptography.Primitives/src/System/Security/Cryptography/CipherMode.cs

```diff
@@ -2,18 +2,21 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Diagnostics;
+using System.ComponentModel;
 
 namespace System.Security.Cryptography
 {
-    // This enum represents cipher chaining modes: cipher block chaining (CBC), 
-    // electronic code book (ECB), and ciphertext-stealing (CTS).  Not all implementations 
-    // will support all modes.
+    // This enum represents supported cipher chaining modes:
+    //  cipher block chaining (CBC),
+    //  electronic code book (ECB),
+    //  ciphertext-stealing (CTS).
+    // Not all implementations will support all modes.
     public enum CipherMode
     {
         CBC = 1,
-        CTS = 5,
         ECB = 2,
+        [EditorBrowsable(EditorBrowsableState.Never)]OFB = 3,
+        [EditorBrowsable(EditorBrowsableState.Never)]CFB = 4,
```

Comment:
It looks like AESManaged rejects these as well. Not sure if we'd want to try to port that.